app/assets/stylesheets/components/_sortable-guiders.scss

```diff
@@ -0,0 +1,3 @@
+.sortable-guiders.changes {
+  margin-bottom: 38px;
```

Comment:
I've removed this - don't think it's worth doing really as you can still see the last item and drag it